docs/UserGuide.md

```diff
@@ -155,6 +170,36 @@ Examples:
 `edit 1 ct/91234567 e/grab@example.com` edits the contact and email of the 1st application to be 91234567 and grab@example.com respectively.
 `edit 2 c/Garena t/` edits the company of the 2nd application to be `Garena` and clears all existing tags.
 
+### Archiving an internship application: `archive`
+
+Archives the specified internship application from the list.
+
+Format: `archive INDEX`
+
+* Archives the internship application at the specified `INDEX`.
+* The application specified must be not archived.
+* The index refers to the index number shown in the displayed internship application list.
+* The index **must be a positive integer** 1, 2, 3, …​
+* The index must be within the range of available internships in the shown application list.
+
+Examples:
+* `list` followed by `archive 2` archives the 2nd internship application in the list.
+
+### Retrieving an internship application from archives: `retrieve`
+
+Retrieves the specified internship application from the list.
+
+Format: `retrieve INDEX`
+
+* Retrieves the internship application at the specified `INDEX`.
+* The application specified must be archived, `list-archive` command is recommended to be used before using this command.
```

Comment:
recommended, just that if archiving one application that has already been archived will throw command exception. The exception will also tell user to use `list-archive`. 

Considering user will also have chance in to use archive after find command, so it doesn't mean that it will always throw command exception if `list-archive` is not used.

Reply:
Yup, because you are retrieving the application that is not archived. 

So usually `list-archive` should be use in order to ensure that the application shown in the UI are all archived, but considering if user using find command which will also possible to search for archived application and show on the UI, I put "recommended" instead of "must" in the user guide.

Reply:
okay got it! 👍